3-async-await/solution.js

```diff
@@ -22,14 +22,62 @@ Example:
 
 function solution() {
     // YOUR SOLUTION GOES HERE
+    const products = require('./products');
+    const prices = require('./prices');
 
     // You generate your id value here
+    const id = Number((Date.now()+'').slice(-2));
+    const result = {id};
 
-    // You use Promise.all() here
+    (async () => {
+        // You use Promise.all() here
+        console.log('Promise.all:\n')
 
-    // You use Promise.allSettled() here
+        try {
+            [result.product, result.price] = await Promise.all([products(id), prices(id)]);
```

Comment:
initialize other constants to get the results here
```
const [a, b] = await Promise.all()
```

Reply:
ok, that way it's more readable